firstvoices/firstvoices/settings.py

```diff
@@ -137,7 +137,10 @@
 
 JWT = jwt.config()
 
-CORS_ALLOWED_ORIGINS = ["http://localhost:3000", os.getenv("ALLOWED_ORIGIN")]
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:3000",
+    os.getenv("ALLOWED_ORIGIN", default="http://localhost:3000"),
```

Comment:
+1 for this addition!

Reply:
Oops, I actually committed this accidentally. @plasticviking Would love your thoughts on a sensible default here